packages/shared/constants.ts

```diff
@@ -0,0 +1,3 @@
+// The PageStyle actor flattens the DOM CSS objects a little bit, merging Rules and their Styles into one actor.
+// For elements (which have a style but no associated rule) we fake a rule with the following style id.
+export const ELEMENT_STYLE = 100;
```

Comment:
wow this is weird. I had not seen this yet. Given that this will now be the `git blame` for this line, it might be worth throwing in a better `sha` to check for history, or just a pointer to somewhere else to learn more?

Reply:
There was no original context. See commit 171ea70586c4b26e2509c0698cde1e25636af1cb with message:
> Add inspector client